build.rs

```diff
@@ -6,6 +6,7 @@ use syntect::highlighting::ThemeSet;
 use syntect::parsing::SyntaxSetBuilder;
 
 fn main() {
+    println!("cargo:rerun-if-changed=assets");
```

Comment:
This wouldn't allow us to detect modified files inside the directory. Instead, we would need to print a line for each file in the directory.

See https://github.com/rust-lang/cargo/issues/2599

Reply:
Ah, yeah.

Better than nothing, though. I expect modifying files will be rare compared to adding and removing them.

Reply:
It could be annoying when someone tries to modify a theme file and nothing happens. We could add a comment or even better, iterate over syntax and theme files inside the assets directory.